Library/Environment.swift

```diff
@@ -121,7 +124,8 @@ public struct Environment {
     reachability: SignalProducer<Reachability, NoError> = Reachability.signalProducer,
     scheduler: DateScheduler = QueueScheduler.main,
     ubiquitousStore: KeyValueStoreType = NSUbiquitousKeyValueStore.default,
-    userDefaults: KeyValueStoreType = UserDefaults.standard) {
+    userDefaults: KeyValueStoreType = UserDefaults.standard,
+    variables: EnvironmentVariables = EnvironmentVariables()) {
```

Comment:
Could `envVariables` or `environmentVariables` be a more descriptive name for this argument?

Reply:
My initial thinking was that since this is on an `Environment` object it already has a context that these are environment variables `Environment.variables`. The way this is called is `AppEnvironment.current.variables` so maybe naming this `environmentVariables` doesn't sound so bad after all! I can make the change!